### PR TITLE
UG DOCUMENTATION 831881:Prepare ItemReordering UG in .NET MAUI ListView

### PR DESCRIPTION
**Pull request by AnanthaLakshmiKannan**
Updated the review correction.

### CODE REVIEW

**Review by Lakshminatarajan — APPROVED** [review]
Approved

File: MAUI/Accordion/getting-started.md [hunk 1]
```diff
@@ -266,20 +260,20 @@ The `SfAccordion` allows you to customize the vertical spacing between the accor
 
 There are four built-in events in the SfAccordion control namely:
 
-* Expanding
-* Expanded
-* Collapsing
-* Collapsed
+* [Expanding](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.Accordion.SfAccordion.html#Syncfusion_XForms_Accordion_SfAccordion_Expanding)
```
Comment (Lakshminatarajan) [inline MAUI/Accordion/getting-started.md:263]: Provided links are Xamarin APIs. Please update with MAUI link.

File: MAUI/Accordion/getting-started.md [hunk 2]
```diff
@@ -266,20 +260,20 @@ The `SfAccordion` allows you to customize the vertical spacing between the accor
...
+* [Expanded](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.Accordion.SfAccordion.html#Syncfusion_XForms_Accordion_SfAccordion_Expanded)
+* [Collapsing](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.Accordion.SfAccordion.html#Syncfusion_XForms_Accordion_SfAccordion_Collapsing)
+* [Collapsed](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.Accordion.SfAccordion.html#Syncfusion_XForms_Accordion_SfAccordion_Collapsed)
 
 ### Expanding Event
 
-The `Expanding` event will be triggered when the accordion item is being expanded. It can cancel expansion with `ExpandingAndCollapsingEventArgs` that contains the following property:
+The [Expanding](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.Accordion.SfAccordion.html#Syncfusion_XForms_Accordion_SfAccordion_Expanding) event will be triggered when the accordion item is being expanded. It can cancel expansion with [ExpandingAndCollapsingEventArgs](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.Accordion.ExpandingAndCollapsingEventArgs.html) that contains the following property:
```
Comment (Lakshminatarajan) [inline MAUI/Accordion/getting-started.md:270]: Provided links are Xamarin APIs. Please update with MAUI link.

File: MAUI/Accordion/getting-started.md [hunk 3]
```diff
@@ -328,13 +322,13 @@ private void SfAccordion_Expanded(object sender, Syncfusion.Maui.Accordion.Expan
 
 ### Collapsing Event
 
-The `Collapsing` event will be triggered when the expander control is being collapsed.It can cancel collapsing with `ExpandingAndCollapsingEventArgs` that contains the following property:
+The [Collapsing](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.Accordion.SfAccordion.html#Syncfusion_XForms_Accordion_SfAccordion_Collapsing) event will be triggered when the expander control is being collapsed.It can cancel collapsing with [ExpandingAndCollapsingEventArgs](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.Accordion.ExpandingAndCollapsingEventArgs.html) that contains the following property:
```
Comment (Lakshminatarajan) [inline MAUI/Accordion/getting-started.md:325]: Provided links are Xamarin APIs. Please update with MAUI link.

File: MAUI/Accordion/getting-started.md [hunk 4]
```diff
@@ -358,10 +352,10 @@ private void SfAccordion_Collapsing(object sender, Syncfusion.Maui.Accordion.Exp
 
 ### Collapsed Event 
 
-The `Collapsed` event is triggered when the accordion item is collapsed. You can execute your own code when this event occurs.
+The [Collapsed](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.Accordion.SfAccordion.html#Syncfusion_XForms_Accordion_SfAccordion_Collapsed) event is triggered when the accordion item is collapsed. You can execute your own code when this event occurs.
```
Comment (Lakshminatarajan) [inline MAUI/Accordion/getting-started.md:355]: Provided links are Xamarin APIs. Please update with MAUI link.

File: MAUI/Accordion/getting-started.md (hunk 3 above)
Comment (AnanthaLakshmiKannan) [reply MAUI/Accordion/getting-started.md]: Addressed the concern provided

File: MAUI/Accordion/migration.md [hunk 5]
```diff
@@ -30,30 +30,39 @@ N> `DynamicSizeMode` property from the Xamarin SfAccordion is not moved to the M
...
+
 <tr>
-<td>IconColor</td>
-<td>HeaderIconColor</td>
-<td>Gets or sets the color of the header icon in the Accordion control.</td></tr>
-</table>
+<td>{{'[IconColor](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.Accordion.AccordionItem.html#Syncfusion_XForms_Accordion_AccordionItem_IconColor)'| markdownify }}</td>
+<td>{{'[HeaderIconColor](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Accordion.AccordionItem.html#Syncfusion_Maui_Accordion_AccordionItem_HeaderIconColor)'| markdownify }}</td>
+<td>Gets or sets the color of the header icon in the Accordion control.</td>
+</tr>
```
[the remark this answers is not in the document]
Comment (AnanthaLakshmiKannan) [reply MAUI/Accordion/migration.md:46]: Addressed the concern provided

File: MAUI/Accordion/getting-started.md (hunk 4 above)
Comment (AnanthaLakshmiKannan) [reply MAUI/Accordion/getting-started.md]: Addressed the concern provided

File: MAUI/Accordion/getting-started.md (hunk 2 above)
Comment (AnanthaLakshmiKannan) [reply MAUI/Accordion/getting-started.md]: Addressed the concern provided

File: MAUI/Accordion/getting-started.md (hunk 1 above)
Comment (AnanthaLakshmiKannan) [reply MAUI/Accordion/getting-started.md]: Addressed the concern provided

**Review by Lakshminatarajan — APPROVED** [review]
Approved

**Review by PiruthivirajM — APPROVED** [review]
approved

**Pull request merged**